crates/python/qcs_sdk/_executable.pyi

```diff
@@ -2,81 +2,105 @@
 Do not import this file, it has no exports.
 It is only here to represent the structure of the rust source code 1:1
 """
+
 from enum import Enum
+from typing import Dict, List, Optional, final
 
-from typing import Dict, List, Optional
-from .qpu.quilc import CompilerOpts
+from .compiler.quilc import CompilerOpts
 from ._execution_data import ExecutionData
 
-class QCSExecutionError(RuntimeError):
-    """Error encounteted when executing programs."""
 
+class ExecutionError(RuntimeError):
+    """Error encountered when executing a program."""
     ...
 
+
+@final
```

Comment:
Nice! Never thought to put `@final` on these but they totally should be.

Reply:
`stubtest` insisted I add it 😂